robot_manager:
	ros__parameters:
		joint_damping: [0.7 0.7 0.7 0.7 0.7 0.7 0.7]
		
